wordpress-mobile/gutenberg-mobile
# Use RichText component on Title block

This PR fixes #502 and moves forward #372 by making the title block working fine with the Enter.Key pressed event.
On Enter.Key pressed event, the Keyboard is not dismissed anymore and you can keep writing into the next block.

GB-side PR: https://github.com/WordPress/gutenberg/pull/13548

To test:

- Start the mobile demo app
- Put the caret somewhere in the Title block
- Tap enter on the soft keyboard, or on the hw keyboard
- It should add a new empty (para) block after the Title

**Note**
<del>With this PR the styling of the `Title` block is lost, we will re-add it back on a separate PR when the iOS side of AztecWrapper will support styling. See this PR here: https://github.com/wordpress-mobile/react-native-aztec/pull/114</del>
